cscore/build.gradle

```diff
@@ -38,6 +39,7 @@ ext {
                     }
                     exportedHeaders {
                         srcDirs 'src/main/native/include', 'src/main/native/cpp'
+                        include '**/*.h'
```

Comment:
What does this do?  It doesn't add them to a distribution file right?

Reply:
This makes it so visual studio generation works. Otherwise, it notices multiple source sets include the same files and gets confused. This doesn't change anything about the actual build.